Teams.Tests/Teams.Tests.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.8" />
```

Comment:
here should be reference to MockQueryable.Core and MockQueryable.Moq
First install core